src/entities/protocols/uniswap.ts

```diff
@@ -260,6 +264,10 @@ function addMixedSwap<TInput extends Currency, TOutput extends Currency>(
     }
   }
 
+  if(tradeType !== TradeType.EXACT_INPUT) {
```

Comment:
why is this one only exact in and not exact out too?

Reply:
We only support mixed routes for EXACT_INPUT: https://github.com/Uniswap/router-sdk/blob/main/src/entities/mixedRoute/trade.ts#L200 , not sure exactly why